features/step_definitions/individual_steps.rb

```diff
@@ -1093,11 +1119,12 @@ class VlpDocument
 
 Then(/the individual should see the elected APTC amount and click on the Confirm button of the Thank You page/) do
   wait_for_ajax
-  expect(page).to have_content '$50.00'
+#  expect(page).to have_content '$50.00'
   find(IvlConfirmYourPlanSelection.i_agree_checkbox).click
-  fill_in IvlConfirmYourPlanSelection.first_name, :with => (@u.find :first_name)
-  fill_in IvlConfirmYourPlanSelection.last_name, :with => (@u.find :last_name)
-  screenshot("aptc_purchase")
+  fill_in IvlConfirmYourPlanSelection.first_name, :with => ("Patrick")
+  fill_in IvlConfirmYourPlanSelection.last_name, :with => ("Doe")
+  #screenshot("aptc_purchase")
```

Comment:
remove comments if unneeded steps

Reply:
this is a needed step, good catch! @ATBull81